modules/ROOT/pages/fips-140-2-compliance-support.adoc

```diff
@@ -27,9 +27,17 @@ Mule relies on the Java runtime to provide a FIPS-compliant security module, whi
 
 Details for this process vary according to your selected security provider. Please refer to the documentation for your security provider for complete instructions.
 
-=== Example: Setup Instructions Using Bouncy Castle Security Provider
+=== Examples
 
-The following example demonstrates how to install and configure Bouncy Castle security provider.
+The following examples demonstrates how to set up a FIPS 140-2 Java environment.
+
+* <<example1>>
+* <<example2>>
+
+[[example1]]
+==== Example: Setup Instructions Using Bouncy Castle Security Provider in Java 8
```

Comment:
More than examples, I would call them Steps to setup (one for java 8, the other for java 11+)